### PR TITLE
Add new book by Artur Czemiel

### PR DESCRIPTION
**Pull request by aexol**
I added my own book that was released today

## Description

I added my own book that was released today. It is mostly about GraphQL itself. It also shows how to create backends with GraphQL Yoga and Axolotl and connect them to fronted with Vite, GraphQL, TS, React, Zeus

### CODE REVIEW

**Comment by aexol** [comment]
By the way 
https://www.graphql.college/fullstack-graphql
https://pragprog.com/book/wwgraphql/craft-graphql-apis-in-elixir-with-absinthe
https://www.packtpub.com/web-development/learning-graphql-and-relay

Those links seem to be inactive please correct them

File: src/pages/community/resources/books.mdx [hunk 1]
```diff
@@ -8,3 +8,4 @@
 - [Hands-on Full-Stack Web Development with GraphQL and React](https://www.packtpub.com/web-development/hands-full-stack-web-development-graphql-and-react) by Sebastian Grebe
 - [The Road to GraphQL](https://www.robinwieruch.de/the-road-to-graphql-book/) by Robin Wieruch
 - [Production Ready GraphQL](https://book.productionreadygraphql.com/) by Marc-Andre Giroux
+- [GraphQL Best Practices: Hands-on experience with schema design, security, and error handling for developers](https://www.amazon.com/dp/B0D9H7MJQV?ref=cm_sw_r_cp_ud_dp_95EHRMW4JFXM7ECF9KGF_2&ref_=cm_sw_r_cp_ud_dp_95EHRMW4JFXM7ECF9KGF_2&social_share=cm_sw_r_cp_ud_dp_95EHRMW4JFXM7ECF9KGF_2) by Artur Czemiel, Nov 2024
```
Comment (benjie) [inline src/pages/community/resources/books.mdx]: Removing tracking:

```suggestion
- [GraphQL Best Practices: Hands-on experience with schema design, security, and error handling for developers](https://www.amazon.com/dp/B0D9H7MJQV) by Artur Czemiel, Nov 2024
```

**Review by benjie — APPROVED** [review]
Congratulations on getting the book published!

**Comment by aexol** [comment]
ok I think you can merge

**Comment by aexol** [comment]
@benjie can you merge? BTW if you want free print copy of this book I can give your email to packt so they can send in return for social media shout out or something. Were you on one conference in Wrocław as a speaker on the screen connecting remotely ?

**Comment by benjie** [comment]
@aexol Yes I did speak remotely there once :)

**Pull request merged**